client/www/scripts/modules/common/common.directives.js

```diff
@@ -242,6 +243,55 @@ Common.directive('slPopoverMenu', [
     };
   }]);
 
+Common.directive('slPopoverInfo', [
```

Comment:
can you add the module to the directive name?

makes it easier to find the directive when troubleshooting